tests/rptest/tests/mirror_maker_test.py

```diff
@@ -0,0 +1,194 @@
+                                         target_cluster=self.redpanda)
+        topics = []
+        for i in range(0, 10):
+            topics.append(
+                TopicSpec(partition_count=random.randint(1, 10),
+                          retention_bytes=random.randint(100000000, 300000000),
+                          retention_ms=random.randint(1 * 3600000,
+                                                      10 * 3600000)))
+        self.source_client.create_topic(topics)
+        self.mirror_maker.start()
+        # start source producer & target consumer
+        self.start_workload()
+
+        self.run_validation(consumer_timeout_sec=120)
+        self.mirror_maker.stop()
+        target_client = DefaultClient(self.redpanda)
+        for t in topics:
+            desc = target_client.describe_topic(t.name)
+            self.logger.debug(f'source topic: {t}, target topic: {desc}')
+            assert len(desc.partitions) == t.partition_count
+
+    @cluster(num_nodes=9)
+    @parametrize(source_type=kafka_source)
+    @parametrize(source_type=redpanda_source)
+    def test_consumer_group_mirroring(self, source_type):
+        # start redpanda
+        self.start_brokers(source_type=source_type)
+        consumer_group = "test-group-1"
+        # start mirror maker
+        self.mirror_maker = MirrorMaker2(self.test_context,
+                                         num_nodes=1,
+                                         source_cluster=self.source_broker,
+                                         target_cluster=self.redpanda,
+                                         consumer_group_pattern=consumer_group,
+                                         log_level="TRACE")
+        self.mirror_maker.start()
+
+        msg_cnt = 100
+        # produce some messages to source redpanda
+        producer = RpkProducer(self.test_context,
+                               self.source_broker,
+                               self.topic.name,
+                               512,
+                               msg_cnt,
+                               acks=-1)
+
+        producer.start()
+        producer.wait()
+        producer.free()
+
+        # consume some messages from source redpanda
+        consumer = RpkConsumer(self.test_context,
+                               self.source_broker,
+                               self.topic.name,
+                               ignore_errors=False,
+                               retries=3,
+                               group=consumer_group,
+                               num_msgs=20)
+
+        consumer.start()
+        consumer.wait()
+        consumer.stop()
+        source_messages = consumer.messages
+        self.logger.info(f"source message count: {len(source_messages)}")
+        consumer.free()
+
+        src_rpk = RpkTool(self.source_broker)
+        source_group = src_rpk.group_describe(consumer_group)
+        # consume single message on target node to create consumer group
```

Comment:
I think instead of creating the consumer group on the target manually you can try to lower the refresh.groups.interval.seconds property (default to 600 secs), so that MM2 does the group creation for you

Reply:
mm2 does not create the group, there is no Kafka API to do that really, i am already using 10s for offset commit sync intervals. MM2 just replicates committed offsets. `refresh.groups.interval.seconds` is responsible for refreshing consumer groups on the source broker.

Reply:
MM2 is indeed creating a group, thank you for the tip